bin/reset-local-dev

```diff
@@ -66,30 +66,33 @@ helm install -f "$CONFIG_FILE" \
   "$GIT_ROOT"
 
 
-echo '
-━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+cat <<EOF
+━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 
- Please run the following command to configure kubectl:
+  Astronomer is launching and should be available shortly. Run the following command
+  to configure kubectl to access the kind cluster that Astronomer is running in:
 
-     export KUBECONFIG="$(kind get kubeconfig-path --name=kind)"
+    kind export kubeconfig
 
-'
-echo "
- Astronomer is launching. You can check the status with:
+  You can then check the status with:
 
-     kubectl get pods -n ${NAMESPACE}
+    kubectl get pods -n ${NAMESPACE}
 
+  After Astronomer is up (all pods running, all containers up), find the root login
+  credentials:
 
- After Astronomer is up (all pods running, all containers up), do this to start a connection to the service:
+    kubectl -n ${NAMESPACE} get secrets astronomer-root-admin-credentials \\
+      -o go-template --template='{{.data.password|base64decode}}{{\"\n\"}}'
 
-     sudo -E kubectl port-forward -n ${NAMESPACE} svc/astronomer-nginx 443
+  Next, port-forward into the kind cluster so you can access the application:
 
+    sudo -E kubectl port-forward -n ${NAMESPACE} svc/astronomer-nginx 443
 
- Then you can access the service here: https://app.localtest.me
+  You can then access the service here: https://app.localtest.me
```

Comment:
Is it a service, or an application? You're using the terms interchangeably. Select one term and use it consistently.

Reply:
In this context, the user is looking at a distributed system which constitutes an application. One of those components is a service that the user accesses to get the web UI, but that is not the only part of the application the user cares about.